packages/mantine/src/components/collection/Collection.tsx

```diff
@@ -139,7 +175,7 @@ const defaultProps: Partial<CollectionProps<unknown>> = {
     getItemId: ({id}: any) => id,
 };
 
-export const Collection = <T,>(props: CollectionProps<T>) => {
+export const Collection = <T,>(props: CollectionProps<T> & {ref?: ForwardedRef<HTMLDivElement>}) => {
```

Comment:
I think we need to call `forwardRef()` here otherwise you cannot pass in a ref as prop